health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/L2Main.js

```
@@ -0,0 +1,757 @@
+function getProjectTypeFromURL() {
+  const url = window.location.pathname;
+  const projectTypes = Digit.SessionStorage.get("projectTypes");
+  const matchingProject = projectTypes?.find(
+    (item) => item?.dashboardUrls && Object.values(item?.dashboardUrls)?.some((dashboardUrl) => url === dashboardUrl)
+  );
+
+  // Return the id of the matching object or null if not found
+  const projectTypeCode = matchingProject ? matchingProject.code : null;
+  return projectTypeCode;
+}
+const L2Main = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)